modal.js

```diff
@@ -45,7 +45,7 @@ export class Modal {
   }
 }
 
-/* FETCH API BOOKING 1 */
+/* FETCH API BOOKING 1 and 2 */
```

Comment:
When I'm reading this, I'm thinking: What does "1 and 2" mean?

I'm guessing that it's something related to your group's process, i.e. how you've divided tasks amongst yourselves or something like that. But that's not relevant to the code, and when someone (e.g. future you) reads this again in six months or a year, you probably will also wonder what "1 and 2" means.

This is another example of how comments can sometimes be more confusing than helpful.